### PR TITLE
No parent poms

### PR DESCRIPTION
**Pull request by alexsuter**

### CODE REVIEW

File: demos-app/demos-app/assembly.xml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+	
 	<formats>
 		<format>zip</format>
 	</formats>
 	<includeBaseDirectory>false</includeBaseDirectory>
 
 	<dependencySets>
 		<dependencySet>
-			<useProjectArtifact>true</useProjectArtifact>
+			<useProjectArtifact>false</useProjectArtifact>
```
Comment (ivy-lli) [inline demos-app/demos-app/assembly.xml:12]: isn't here true important, so it packs all the other demo projects?

File: html-dialog/html-dialog-demos-test/pom.xml [hunk 2]
```diff
@@ -19,33 +30,39 @@
       <version>${project.version}</version>
       <type>iar</type>
     </dependency>
+    <dependency>
+      <groupId>com.axonivy.ivy.webtest</groupId>
+      <artifactId>web-tester</artifactId>
+      <version>9.2.1</version>
```
Comment (ivy-lli) [inline html-dialog/html-dialog-demos-test/pom.xml:36]: maybe use version 9.3.0 if possible?

**Review by ivy-lli — APPROVED** [review]
Looks good 🥳. Two small comments...

File: html-dialog/html-dialog-demos-test/pom.xml (hunk 2 above)
Comment (alexsuter) [reply html-dialog/html-dialog-demos-test/pom.xml:36]: tests are failing then.

Comment (ivy-lli) [reply html-dialog/html-dialog-demos-test/pom.xml:36]: ok

File: demos-app/demos-app/assembly.xml (hunk 1 above)
Comment (alexsuter) [reply demos-app/demos-app/assembly.xml:12]: Tested on my machine. Seems to work smoothly.

**Pull request merged**